modules/sandboxed-containers-deleting-kataconfig-crd-web.adoc

```diff
@@ -0,0 +1,24 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+:_content-type: PROCEDURE
+[id="sandboxed-containers-deleting-kataconfig-crd-web_{context}"]
+= Deleting the `KataConfig` custom resource definition using the web console
+
+The `KataConfig` custom resource definition (CRD) lets you define the `KataConfig` CR. To complete the uninstall process, delete the `KataConfig` CRD from your cluster.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have removed the `KataConfig` CR from your cluster.
+* You have removed the {sandboxed-containers-operator} from your cluster.
+
+.Procedure
+
+. From the *Administrator* perspective, navigate to *Administration* → *CustomResourceDefinitions*.
```

Comment:
Not sure but does "from the administrator perspective" mean they have to be logged in as the administrator?

Reply:
Not exactly. They log in and then select Administrator rather than User from the drop down list. I used the same terminology as elsewhere in the section, though mode would also work if preferable.